### ISSUE TITLE
THREE.ObjectLoader and Textures

### ISSUE THREAD

**Issue by laserallan** [mrdoob/three.js#4967]
I'm trying to load a scene a material texture using THREE.ObjectLoader() and I can't get it to work. It seems like specifying a map doesn't make any difference and it always comes out as null. Is it supported?
This is the basic form for my datafile

``` json
{
   "geometries":[
      {
         "data":{
            "faces":[...],
            "normals":[...],
            "uvs":[...],
            "vertices":[...]
         },
         "type":"Geometry",
         "uuid":"33628ef2-5a1f-4f4c-8cd2-c717c512137c"
      }
   ],
   "images":[
      {
         "url":"myimage.png",
         "uuid":"13721d96-17ae-485b-bfdf-f27fcd280e97"
      }
   ],
   "materials":[
      {
         "ambient":16755455,
         "color":16777215,
         "emissive":0,
         "map":"9aee2d5f-b760-4449-adbb-0baad2d78b9a",
         "name":"DummyMaterial",
         "opacity":1,
         "shininess":1,
         "specular":0,
         "transparent":false,
         "type":"MeshPhongMaterial",
         "uuid":"43806f03-5941-4765-b058-2562c7a2ca2a",
         "wireframe":false
      }
   ],
   "metadata":{
      "generator":"MyGenerator",
      "type":"Object",
      "version":4.3
   },
   "object":{
      "children":[
         {
            "children":null,
            "matrix":[...],
            "type":"Mesh",
            "uuid":"9cf4c9b4-a3bf-40ee-8a4b-b5be861dca84",
            "geometry":"33628ef2-5a1f-4f4c-8cd2-c717c512137c",
            "material":"43806f03-5941-4765-b058-2562c7a2ca2a",
            "name":"StaticMeshActor_0"
         }
      ],
      "matrix":[...],
      "type":"Scene",
      "uuid":"d2e735e9-5487-4d8b-86d3-3863561b3f80"
   },
   "textures":[
      {
         "image":"13721d96-17ae-485b-bfdf-f27fcd280e97",
         "uuid":"9aee2d5f-b760-4449-adbb-0baad2d78b9a"
      }
   ]
}
```

**Comment by kevzettler** [comment]
Any update on this or any alternatives? I exported a whole scene using `scene.toJSON` and have reloaded it with `THREE.ObjectLoader.prototype.parse()` It renders but seems to be missing all the textures. Is that related to this issue?

**Comment by mrdoob** [comment]
>  I exported a whole scene using `scene.toJSON` and have reloaded it with `THREE.ObjectLoader.prototype.parse()` It renders but seems to be missing all the textures. Is that related to this issue?

No idea...

**Comment by kevzettler** [comment]
@mrdoob What I'm asking is should I expect them to work? or does this issue imply that it won't. the scene JSON export has a `materials` list that looks ok.

**Comment by mrdoob** [comment]
I see... In theory they should work now.

**Comment by Mugen87** [comment]
Should be solved via #5502
